File: backend/src/main/java/com/databaseproject/backend/request/UserRequest.java
package com.databaseproject.backend.request;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@Data
@ToString
public class UserRequest {
    private String username;
    private String password;
    private String firstName;
    private String lastName;
    private String email;
    private String phone;
    private String address;
}

File: Front End/online-bookstore/src/app/shared/UserRequest.ts
export class UserRequest{   
     username:string=""
     password:string="";
     firstName:string=""
     lastName:string=""
     email:string=""
     phone:string=""
     address:string=""

     fillData(user:string,pass:string,first:string,last:string,mail:string,pho:string,add:string){
        this.username=user
        this.password=pass
        this.firstName=first
        this.lastName=last
        this.email=mail
        this.phone=pho
        this.address=add
     }

}